src/components/pages/career-details/career-form/career-form.tsx

```diff
@@ -32,15 +32,25 @@ interface CareerFormProps {
   scrollToStart?: () => void;
 }
 
-interface RawFormData {
+interface CategorySelect {
+  [key: string]: string;
+}
+
+export type CareerFormValues = {
```

Comment:
I don't particularly like the plural here. It's only a single value in my opinion.

Reply:
+1

Plural is an array, this is an interface/object.

Reply:
good point, I will fix that.

Reply:
Oh I see where my mind came from: `FieldValues` is the internal default. Anyway, I updated this.